src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -181,6 +187,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     Outputs="$(_GeneratedStaticWebAssetsDevelopmentManifest)"
     DependsOnTargets="$(GenerateStaticWebAssetsManifestDependsOn)">
 
+    <ItemGroup>
+      <_WebRootFiles Include="@(ContentWithTargetPath)" Condition="$([System.String]::Copy('%(TargetPath)').StartsWith('wwwroot'))" />
```

Comment:
Does this need to account for things like `TargetPath=wwwroot2/File.txt`?

Reply:
@javiercn You are right, we can do "wwwroot/" instead. It'll be safer.